bootstrap.sh

```
@@ -213,27 +277,27 @@ echo "--------------------------------------------------------------------------
 if [ -x "$(command -v timedatectl)" ]; then
   runCmdAndLog timedatectl set-timezone ${TIMEZONE}
 fi
 
 # Locales
-log "Setting locales to ${LOCALES[*]}..."
+logStep "Setting locales to ${LOCALES[*]}..."
 LOCALE_GEN=locale-gen
 if ! [ -x "$(command -v ${LOCALE_GEN})" ]; then
   runCmdAndLog ${APT_INSTALL} locales
 fi
 runCmdAndLog ${LOCALE_GEN} ${LOCALES[@]}
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)